clients/web/src/views/layout/RegisterView.js

```diff
@@ -75,10 +75,6 @@ var RegisterView = View.extend({
 
             this.$('#g-register-button').girderEnable(false);
             this.$('.g-validation-failed-message').text('');
-        },
-
-        'click a.g-login-link': function () {
```

Comment:
I don't think you want to be removing this, since you're not updating the template for *this* particular view.